### DISCUSSION TITLE
Representing Medicare Pricing methodologies for in-network file

### DISCUSSION THREAD

**Discussion by iGokul-G** [CMSgov/price-transparency-guide#31] [General]
How do we represent payment methodologies like a percentage of Medicare APC / ASC. Typically these products are supplied by vendors which may or may not include Grouper editor and Pricer. 

Example : Hosp ABC contract may say pay X % of Medicare APC for Outpatient services

**Comment by ghost** [comment]
We are looking at generating a "dummy claim" to provide the associated negotiated price for the billing code to obtain the required dollar amount for the negotiated price field (since it requires a dollar amount).
